packages/components/src/schemas/components.ts

```diff
@@ -26,6 +26,7 @@ import {
   UnorderedListSchema,
   VideoSchema,
 } from "~/interfaces"
+import { LogoCloudSchema } from "~/interfaces/complex/LogoCloud"
```

Comment:
nit - can be `"~/interfaces` and merge with the above instead of `~/interfaces/complex/LogoCloud`

Reply:
thanks for the catch!